README.md

```diff
@@ -72,14 +72,16 @@ jobs:
-| service_url      | ✔        | https://components.espressif.com/api | (Deprecated) IDF Component registry API URL                                                                                      |
-| registry_url     | ✔        | https://components.espressif.com/    | IDF Component registry URL                                                                                                       |
+| Input            | Optional | Default                              | Description                                                                                                                       |
+|------------------|----------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| api_token        | ❌        |                                      | API Token for the component registry                                                                                              |
+| namespace        | ❌        |                                      | Component namespace                                                                                                               |
+| name             | ✔ / ❌    |                                      | Name is required for uploading a component from the root of the repository                                                        |
+| version          | ✔        |                                      | Version of the component, if not specified in the manifest. Should be a [semver](https://semver.org/) like `1.2.3` or `v1.2.3`    |
+| directories      | ✔        | Repo root                            | Semicolon separated list of directories with components.                                                                          |
+| skip_pre_release | ✔        | False                                | Set this flag to `true`, `t`, `yes` or `1` to skip [pre-release](https://semver.org/#spec-item-9) versions.                       |
+| dry_run          | ✔        | False                                | Set this flag to `true`, `t`, `yes` or `1` to upload a component for validation only without creating a version in the registry.  |
+| service_url      | ✔        | https://components.espressif.com/api | (Deprecated) IDF Component registry API URL                                                                                       |
+| registry_url     | ✔        | https://components.espressif.com/    | IDF Component registry URL                                                                                                        |
+| repository_url   | ✔        |                                      | URL of the repository where component is located                                                                                  |
```

Comment:
@magicarm22 Wouldn't be better to add these details by default and add a way to disable?

If it's not the default, then it's worth adding these values to the examples above.

Reply:
@magicarm22 Since we support uploading from a directory, it would be nice to track the path and have it in the link in the registry

Reply:
It would be nice to add a switch to send/not-send the source directory

Reply:
@kumekay I updated the default values, PTAL

Tested action: https://github.com/magicarm22/example_component/actions/runs/7917138414/job/21612561022